lib/AST/UnqualifiedLookup.cpp

```diff
@@ -178,6 +178,7 @@ namespace {
     const bool isOriginallyTypeLookup;
     const NLOptions baseNLOptions;
     // Transputs
+    DeclContext *capturedSelfContext;
```

Comment:
Is this an appropriate place for this? It's definitely not an input or an output...

Reply:
Yeah that's correct. Will do.